amlb/benchmarks/openml.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from typing import List, Tuple, Optional
+
+import openml
+
+from amlb.utils import Namespace
+
+
+log = logging.getLogger(__name__)
+
+
+def is_openml_benchmark(benchmark: str) -> bool:
+    """ Check if 'benchmark' is a valid identifier for an openml task or study. """
+    if len(benchmark.split('/')) == 3:
+        domain, oml_type, oml_id = benchmark.split('/')
+        supported_types = ['s', 't']
+
+        valid_id = False
+        try:
+            _ = int(oml_id)
+            valid_id = True
+        except ValueError:
+            pass
+
+        return domain == "openml" or oml_type in supported_types and valid_id
```

Comment:
why do you have a `or` here?
should it not be all `and`?

Reply:
good catch